Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_description.md

```diff
@@ -33,7 +33,4 @@ If MFA is enabled for your user, follow the next steps:
 3. **look-back parameter note**:
 In case the **look-back** parameter is initialized with a certain value and during a time that incidents were fetched, if changing 
 the look back to a number that is greater than the previous value, then in the initial incident fetching there will be incidents duplications.
-If the integration was already set with look back > 0, and the look-back is not being increased at any point of time, then those incident duplications would not occur.
-
----
-[View Integration Documentation](https://xsoar.pan.dev/docs/reference/integrations/service-now-v2)
```

Comment:
Hi @Shellyber I never added this change. I submitted it through marketplace contribution. Must have somehow crept it. Let me recheck it. I also believe there was a SNow content pack update since I duplicated this integration to work on. Let me double check on this.